pyanaconda/ui/tui/tuiobject.py

```diff
@@ -55,3 +86,223 @@ def has_help(self):
     def refresh(self, args=None):
         """Put everything to display into self.window list."""
         UIScreen.refresh(self, args)
+
+
+class AskUserInput(object):
```

Comment:
Names of attributes and variables for instances of the class `AskUserInput`:

```
ask_object
input_handler
ask_input
_nfs_server_input
_nfs_opts_input
hostname_dialog
```
I think that the name of this class is very unfortunate and confusing. I would suggest to use the name `Dialog`, because it is actually used to have a conversation with the user. Or we can use the original `Entry`. Then the variables could have names `something_dialog`, so it is clear what objects they hold.

Reply:
I like the `Entry` and thanks, I wasn't able to came with something sensible.